src/main/kotlin/com/sourcegraph/cody/history/HistoryTree.kt

```diff
@@ -78,6 +78,17 @@ class HistoryTree(
         for (child in sorted) period.add(child)
         model.reload(period)
       }
+    } else {
+      val period =
+          root
+              .periods()
+              .flatMap { it.leafs() }
+              .find { it.chat.internalId == chat.internalId }!!
+              .parent as PeriodNode
+      val sorted = period.leafs().sortedByDescending { it.chat.getUpdatedTimeAt() }
```

Comment:
I'm not sure if that logic is correct. What if you will update chat from different period, e.g. from yesterday?

Reply:
Tip: You probably know this, but for the record: You can easily reproduce this by editing `cody_history.xml` and changing `updatedAt` to some old date.